smash/core/signal_analysis/signatures/_standardize.py

```diff
@@ -47,18 +55,56 @@ def _standardize_signatures_sign(sign: str | list[str] | None) -> tuple[list[str
     return (cs, es)
 
 
-def _standardize_signatures_event_seg(event_seg: dict | None) -> dict:
+def _standardize_compute_signatures_domain(domain: str) -> str:
+    if isinstance(domain, str):
+        domain_standardized = domain.lower()
+
+        if domain_standardized in DOMAIN:
+            domain_standardized = domain_standardized[:3]
+
+        else:
+            raise ValueError(f"Unknown domain argument {domain}. Choices: {DOMAIN}")
+    else:
+        raise TypeError(f"domain argument must be str")
+
+    return domain_standardized
+
+
+def _standardize_compute_signatures_event_seg(event_seg: dict | None) -> dict:
     if event_seg is None:
         event_seg = {}
 
+    elif isinstance(event_seg, dict):
+        if "peak_quant" in event_seg:
+            event_seg["peak_quant"] = _standardize_hydrograph_segmentation_peak_quant(
+                event_seg["peak_quant"]
+            )
+
+        if "max_duration" in event_seg:
+            event_seg[
+                "max_duration"
+            ] = _standardize_hydrograph_segmentation_max_duration(
+                event_seg["max_duration"]
+            )
+
+        if "by" in event_seg:
+            event_seg["by"] = _standardize_hydrograph_segmentation_by(event_seg["by"])
+
+    else:
+        raise TypeError(f"event_seg argument must be None or a dictionary")
+
```

Comment:
```suggestion
    else:
        if isinstance(event_seg, dict):
            if "peak_quant" in event_seg:
                event_seg["peak_quant"] = _standardize_hydrograph_segmentation_peak_quant(
                    event_seg["peak_quant"]
                )
    
            if "max_duration" in event_seg:
                event_seg[
                    "max_duration"
                ] = _standardize_hydrograph_segmentation_max_duration(
                    event_seg["max_duration"]
                )
    
            if "by" in event_seg:
                event_seg["by"] = _standardize_hydrograph_segmentation_by(event_seg["by"])
    
        else:
            raise TypeError(f"event_seg argument must be a dictionary")
```
Il me semble qu'on faisait comme ca dans d'autres standardize et qu'on renvoie pas de type error sur du None